tempesta_fw/http_parser.c

```diff
@@ -297,19 +339,30 @@ __FSM_STATE(st) {							\
 	TFW_PARSER_BLOCK(st);						\
 }
 
+#define __FSM_I_TX_COND_nofixup(st, condition, st_next)			\
+__FSM_STATE(st) {							\
+	if (likely(condition))						\
+		__FSM_I_MOVE_nofixup_n(st_next, 1);			\
+	return CSTR_NEQ;						\
+}
+
```

Comment:
I am not sure this is the right place for this macro, as well as for `__FSM_I_TX_nofixup()` and `__FSM_I_TX_LC_nofixup()` below. All interior macros are grouped together in a section of the source file. I would say that these macros should be defined there as well.